crates/runner/src/lib.rs

```diff
@@ -52,6 +62,11 @@ impl Default for Builder {
             preload: None,
             command: JavyCommand::Build,
             redirect_stdout_to_stderr: None,
+            javy_stream_io: None,
```

Comment:
Not really related to this PR, but for future simplicity, but might be worth it to have the `Builder` autogenerated using [`derive-builder`](https://docs.rs/derive_builder/0.20.1/derive_builder/)